src/emu/cpu/src/arm_dynarmic.cpp

```diff
@@ -74,7 +74,7 @@ namespace eka2l1::arm {
         }
 
         CallbackOrAccessTwoWords CompileGetTwoWords(bool two, unsigned opc, coproc_reg CRm) override {
-            return CallbackOrAccessTwoWords{};
+            return CompileSendTwoWords(two, opc, CRm);
```

Comment:
This too. Don't change meaning where not needed.

Reply:
@pent0 fix it https://github.com/EKA2L1/EKA2L1/commit/e69502495791ac4f7c99fa21d58ad16deb6e435e